ocrd_webapi/main.py

```diff
@@ -50,6 +56,13 @@ async def startup_event():
     """
     await initiate_database(DB_URL)
 
+    # If the default user account is not available
+    # in the DB, create it
+    try:
+        await authenticate_user('test', 'test')
```

Comment:
Use `OCRD_WEBAPI_{USERNAME,PASSWORD}` here? Also a `XXX` comment might be good so we do not forget about the `test` user being created on the fly.

Reply:
I will add a TODO comment to remove it. I also thought about that but many users may be created on the fly when the environment variables change unless the default user account is terminated when the server shuts down.